src/components/Projects/index.tsx

```diff
@@ -287,6 +274,18 @@ export default function Projects() {
           <TrendingProjects count={12} trendingWindowDays={7} />
         </div>
       ) : null}
+      {Boolean(searchText) && !isLoading && (
```

Comment:
**question** is there a particular preference for this in the project? I am a bigger fan of the double bang (I think the airbnb standard also prefers it). I have seen both used in project

Reply:
Heh, I'm the opposite - Boolean() clear

Reply:
Ultimately: it's whatever the linter allows. Lets lint for this if we feel strongly